### PR TITLE
Update some icons and add QtQuick

### PR DESCRIPTION
**Pull request by cgfrost**
I've added some extra examples, the qml icon used for QtQuick files. I've also changed some defaults as I thought they could be more descriptive. Feel free to reject them if you prefer the existing defaults.

### CODE REVIEW

**Comment by ivogabe** [comment]
Thanks for the PR. I'd like to use Ionicons for all icons. (In the future I might add different icon 'packs', which will be based on different icon fonts. Currently other icon packs are included for users to easily customize certain icons.) Can you revert the changes to the icons from devicons? About the JavaScript icon, in my opinion `ion-social-javascript` doesn't fit with the other icons. The color change (.es6) is ok to me.

**Comment by cgfrost** [comment]
Sure, no problem. I'll update the pull request. I use a dark theme the my es6 icons were almost invisible, glad that's OK.

**Comment by cgfrost** [comment]
Everything is using Ionicons now and the JS icons are back to the original defaults. I've updated the readme as well now. Very happy to change anything else you want, thanks for the quick review earlier.

**Comment by cgfrost** [comment]
Actually, I'm being a little lazy. Would you like me to split this out in to separate pull requests?

**Comment by ivogabe** [comment]
A single PR isn't a problem, but can you add icons in different commits? I can then easily choose the commits I want to merge.

Can you revert the ruby icon? The heart looks a bit like the logo of ruby. About the sh and bat icons, I agree that the current isn't the best, but I'm not sure which we should use. `ion-android-desktop` is in my opinion still not the best, maybe `ion-android-list`?

**Comment by cgfrost** [comment]
Hi,

I've split it out in to separate commits and made the changes you asked for. I also agree that ion-android-list is better than my suggestion and I think it's better then the previous choice.

File: main.js [hunk 1]
```diff
@@ -80,6 +80,7 @@ define(function (require, exports, module) {
 	addAlias('class', 'java');
 	addIcon('scala', 'ion-navicon-round file-icon-rotated', '#72d0eb');
 	addIcon('groovy', 'ion-ios-star', '#4298b8');
+    addIcon('mf', 'ion-ios-gear', '#2f5796');
```
Comment (ivogabe) [inline main.js]: Can you fix the indentation here?

File: main.js [hunk 2]
```diff
@@ -110,9 +111,12 @@ define(function (require, exports, module) {
 	addIcon('py', 'ion-social-python', '#f8c63d');
 	addAlias('pyw', 'py');
 
+	// Qt Quick
+	addIcon('qml', 'ion-code', '#42ed0e');
+
 	// Shell and friends
-	addIcon('sh', 'ion-document-text', '#008d00');
```
Comment (ivogabe) [inline main.js]: Can you switch back to the old color? Or, if you have a better color, use that? My design is that extensions in the same 'family' share the same icon, but the icons have different color. For example, all images use ion-image but have different colors.

**Comment by ivogabe** [comment]
Thanks! It's almost good to merge, I found two small things. When you've fixed these, I'll merge in your changes.

**Comment by cgfrost** [comment]
Thanks again for the quick response, I have made the changes/fixes. (Indentation error was a little amateur). Happy to make more changes if you want them.

**Comment by ivogabe** [comment]
Thanks! Merged

**Pull request merged**